packages/core/js-client/src/jsPeer/FluencePeer.ts

```diff
@@ -336,7 +341,8 @@ export abstract class FluencePeer {
           this._incomingParticles.next({
```

Comment:
`_startParticleProcessing` is enormous. More than 300 lines. WDYT about splitting it to a bunch of functions?

Reply:
Added a todo, will do later